### DISCUSSION TITLE
(매우 초보입니다...)유니티 제페토 설명창띄우기

### DISCUSSION THREAD

**Discussion by maskpop** [naverz/zepeto-studio-kor#708] [5. Q&A]
안녕하세요 이번에 유니티를 처음 접하게 되었고 이런저런 고난과 부딪히고 있습니다...
커뮤니티도 둘러보고 유니티 강의도 봤는데 제페토스크립트는 다르게 적용이 되는 것 같아 안되더라구요.. 스크립트 만드는건 아예 처음입니다.
게임을 플레이하면서 탑에 대한 정보를 주기위해서 안내판을 누르면 설명창이 이미지와 텍스트로 카메라 앞에 나올 수 있도록 하고싶은데 언어를 모르고 시간이 없는 저로썬 도무지 해결할 방안이 없습니다... 
쉽게 알 수 있는 설명이 있는 커뮤니티라도 알려주시면 감사하겠습니다...

Uptate()
    {
        this.Mousecontrol();
    }

    Mousecontrol()
    {
        if(Input.GetMouseButtonDown(0)){
            let ray = Camera.main.ScreenPointToRay(Input.mousePosition);
            let ref = $ref<RaycastHit>();

        if(Physics.Raycast(ray, ref, 100))
            {    
             
                let hitInfo = $unref(ref);
    }

안내판을 Player라는 태그를 하고, 그리고 그 태그의 오브젝트를 클릭하면 설명창이 나오게 할 수 있대서 어떻게 하면 좋을지 고민입니다...! 무지해서 죄송합니다..

   float dis = Vector3.Distance(this.gameObject.transform.position, signpan.transform.position);
    if (dis < 3)  //안내판과의 거리
    {
        if(Input.GetMouseButtonDown(0)) //클릭한다면
        {
            panel.SetActive(true); //설명창이 활성화된다
        }
    }

**Comment by x4cf** [comment]
하나용님의 유투브강의
https://www.youtube.com/watch?v=8Sg6Lr3FSII

IT잡동사니님의 유투브강의
https://www.youtube.com/watch?v=CrhExWVP6FI

치자피즈의 유투브강의
https://www.youtube.com/watch?v=U9PNr2VNc34

눈코딩님의 스터디채널
https://www.youtube.com/watch?v=y-gf3HcdEmI&list=PLTFRwWXfOIYCOdgFbutBxPBbvLkb_2GL9&index=27

제페토 공식 멀티 튜토리얼
https://www.youtube.com/watch?v=H92Gd2G9DhM

치자피즈의 가이드, (설명X)
https://cheesafizz.notion.site/1-94030da7011e41d2bc0861c3470795c4

치자피즈의 가이드, (설명O)
https://cheesafizz.notion.site/1-b04332d8a4024f24a49a10b593b03c1c

이런 자료는 어떤가요 :D

**Reply by maskpop** [reply]
몇몇 아는 분들이 계시군용 다른 것들도 다 공부해보겠습니다!! 감사합니다!!
